src/index.css

```
@@ -1,3 +1,220 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+@import url("https://fonts.googleapis.com/css?family=Montserrat:200,300,700");
+
+/* DO THIS BETTER */
+a {
+    text-decoration: none !important;
+  }
```

You just need to be more specific here. Untested code but something like this
```suggestion
a, a:link, a:hover, a:active, a:visited {
    text-decoration: none;
  }
```